mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -0,0 +1,445 @@
+// These examples are from Mapbox (https://www.mapbox.com/mapbox-gl-js/examples/) and therefore under the copyright of Mapbox Inc.
```

Comment:
I don't know what our policy on including copyrighted documentation is. CC @RyanCavanaugh